### PR TITLE
chore!: Change `noir-lang/noir-source-resolver` to `noir-lang/source-resolver`

### PR DESCRIPTION
**Pull request by kevaundray**
# Description

The source-resolver is at version 1.1.3 -- this PR stops updating that package and creates a new package which will synced up with our noir version

## Problem\*

Resolves 

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: compiler/source-resolver/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@noir-lang/noir-source-resolver",
+  "name": "@noir-lang/source-resolver",
   "version": "1.1.4",
```
Comment (TomAFrench) [inline compiler/source-resolver/package.json:3]: Isn't this version supposed to be updated if you want versions to be synced?

File: compiler/wasm/test/shared.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { initialiseResolver } from "@noir-lang/noir-source-resolver";
+import { initialiseResolver } from "@noir-lang/source-resolver";
```
Comment (TomAFrench) [inline compiler/wasm/test/shared.ts:1]:
```suggestion
import { initializeResolver } from "@noir-lang/source-resolver";
```

Doesn't have to be done in this PR but if we're switching the packages then let's yeehawify it.

File: compiler/source-resolver/package.json (hunk 1 above)
Comment (kevaundray) [reply compiler/source-resolver/package.json:3]: I added this package to the release-please-config, so once this gets merged, it should update in the release-please PR

**Review by TomAFrench — APPROVED** [review]
LGTM

File: compiler/wasm/test/shared.ts (hunk 2 above)
Comment (kevaundray) [reply compiler/wasm/test/shared.ts]: just yeehawed the codebase

**Review by TomAFrench — APPROVED** [review]

**Pull request merged**